README.md

```diff
@@ -108,6 +108,8 @@ For an example of a markdown-and-airtable-driven site using `gatsby-transformer-
 
 If you are using the `Attachment` type field in Airtable, you may specify a column name with `fileNode` and the plugin will bring in these files. Using this method, it will create "nodes" for each of the files and expose this to all of the transformer plugins. A good use case for this would be attaching images in Airtable, and being able to make these available for use with the `sharp` plugins and `gatsby-image`. Specifying a `fileNode` does require a peer dependency of `gatsby-source-filesystem` otherwise it will fall back as a non-mapped field. The locally available files and any ecosystem connections will be available on the node as `localFiles`.
 
+When using the `Attachment` type field, requests to download the associated files are limited to 5 concurrent requests to avoid excessive requests to Airtable’s servers resulting in refused connections. This limit can be set with the `concurrency` option in your `gatsby-config.js` file. Set the option with an integer value for your desired limit on attempted concurrent requests. A value of `0` will allow requests to be made with no limit.
```

Comment:
I would update this to read:

When using the `Attachment` type field, this plugin governs requests to download the associated files from Airtable to 5 concurrent requests to prevent excessive requests on Airtable's servers - which can result in refused / hanging connections. You can adjust this limit with the `concurrency` option in your `gatsby-config.js` file. Set the option with an integer value for your desired limit on attempted concurrent requests. A value of `0` will allow requests to be made without any limit.